content/reference/api/http.md

```diff
@@ -1,32 +1,31 @@
----
+----
 title: "HTTP API"
 weight: 20
 menu:
     reference:
         parent: api
----
+----
 
 <!-- TODO: Describe how to change ports and configure the API server -->
-<!-- TODO: Generate newer version of this reference -->
-<!-- TODO: Update generation code to fix links (they've been manually fixed here) -->
-<!-- TODO: Remove index since it's handled by the TOC in the menu? -->
 <!-- TODO: Structure this around command groups (dag, object, files, etc.) -->
 
-<sup>Generated on 2018-06-11, from go-ipfs v0.4.15.</sup>
+<sup>Generated on 2019-01-08, from go-ipfs v0.4.18.</sup>
```

Comment:
I see we missed 3 releases (previous version generated on 2018-06-11, from go-ipfs v0.4.15)
What if regenerating HTTP API docs is a part of Release Dance for go-ipfs?

Reply:
We missed releases because the autogeneration code was broken, because there were major refactoring around commands and I did not want to fix it every second day. But I fixed it now and will do this update more regularly.